tools/deny.toml

```diff
@@ -84,8 +84,8 @@ skip-tree = [
     { name = "windows-sys", version = "=0.42.0" },
 
     # TestSys uses a newer version of base64 and serde_yaml
-    { name = "testsys-model", version = "=0.0.7" },
-    { name = "bottlerocket-types", version = "=0.0.7" },
+    { name = "testsys-model", version = "=0.0.8" },
+    { name = "bottlerocket-types", version = "=0.0.8" },
```

Comment:
Can we skip `base64` and `serde_yaml` individually instead of skipping the entire tree here? I think for `base64` you can just remove the explicit version on L71.

Reply:
I think this might be easier to revisit in #3149 if we want to just merge this for now.